apollo-router/src/services/router/service.rs

```diff
@@ -869,6 +870,10 @@ impl RouterFactory for RouterCreator {
             .for_each(|p| mm.extend(p.web_endpoints()));
         mm
     }
+
+    fn pipeline_ref(&self) -> &PipelineRef {
+        &self._pipeline_handle.pipeline_ref
```

Comment:
Let's remove the leading underscore so it doesn't look like it's unused in the struct definition

Reply:
[55e1b1f](https://github.com/apollographql/router/pull/7009/commits/55e1b1f0a395910346853a35f7d47889165859b5)